src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <div class="form_field">
+        <label class="big_search_wrapper">
+          <img
+            class="icon icon_big"
+            src="images/Search.svg"
+            alt="Search"
+          >
+          <input
+            class="field field_big"
+            type="text"
+            name="SearcBarBig"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+    </form>
+    <form
```

Comment:
```suggestion
    </form>

    <form
```

Add empty line between multiple sibling blocks